fedora-spec-files/Containerfile

```
@@ -0,0 +1,16 @@
+FROM fedora:35
```

```suggestion
FROM fedora:latest
```

Should be safe to go with `latest`

---

I'm sorry but I am not doing that - I don't know what f36 and f37 will look like: they may have completely different package names, dependencies, APIs

---

Given that this is not a production image, I don't think this matters that much: if something brakes, the next user can fix it. But if nothing changes, they can just use this script without any change to the image.

A pinned-down OS version on the other hand can go unnoticed for longer and will certainly need some maintenance work to update it. So for me this boils down to "avoid an issue that might happen" vs "avoid the image getting outdated for sure".

---

I'll change it to :latest, it's not worth for me to argue about this

---

As you wish.